src/util.ts

```diff
@@ -911,14 +911,44 @@ export class Util {
      * If the name does not have a period, and a namespaceName was provided, return the class name prepended by the namespace name.
      * If no namespace is provided, return the `className` unchanged.
      */
-    public getFulllyQualifiedClassName(className: string, namespaceName?: string) {
+    public getFullyQualifiedClassName(className: string, namespaceName?: string) {
         if (className.includes('.') === false && namespaceName) {
             return `${namespaceName}.${className}`;
         } else {
             return className;
         }
     }
 
+    public splitStringIntoLines(string: string) {
```

Comment:
Could we shorten this name a bit, perhaps call it `splitLines` or `splitIntoLines` since we can see it's a `string` based on the parameter type?